packages/kit/src/core/adapt/prerender.js

```diff
@@ -26,6 +26,12 @@ function get_src(attrs) {
 	return match && (match[1] || match[2] || match[3]);
 }
 
+/** @param {string} attrs */
+function is_rel_external(attrs) {
+	const match = /rel\s*=\s*(?:["'][^>]*(external)[^>]*["']|(external))/.exec(attrs);
```

Comment:
this is the kind of thing it'd be really nice to have a unit test for

Reply:
@v1ack can you add a test for this?

Reply:
I added a similar test in https://github.com/sveltejs/kit/pull/1984. That should make it easy for you to get started

Reply:
Sorry for the long wait

Reply:
```suggestion
	return /rel\s*=\s*(?:["'][^>]*(external)[^>]*["']|(external))/.exec(attrs);
```